### ISSUE TITLE
www.naviki.org support

### ISSUE THREAD

**Issue by PiotrPolewczak** [cpfair/tapiriik#331]
hello,
Could You add support for https://www.naviki.org/ ?

**Comment by cpfair** [comment]
They do not have a public API, so no, probably not.

**Comment by PiotrPolewczak** [comment]
Yes, they do not have public api, but endomondo also has no public api, and You have been able to connect with them, so maybe just write them as an application developer and change their mind?

**Comment by cpfair** [comment]
It's more about the effort involved for a relatively unknown fitness tracking site (no offense to Naviki users... but it's no Garmin Connect). If you want to contact them about the API, then write the integration code and submit a PR, I'd be happy to merge that.
